docs/src/main/sphinx/develop/connectors.rst

```diff
@@ -250,6 +250,10 @@ a given query.
 
 Otherwise, these methods return a result object containing a new table handle.
 
+Each of these methods receives the table handle created thus far, and can add
+additional push down information to it by returning a new table handle created
+from the old one.
```

Comment:
This paragraph duplicates the next one and the previous one combined. What are you trying to say here that's not already in the surrounding paragraphs?